ocaml-lsp-server/docs/ocamllsp/typedHoles-spec.md

```diff
@@ -0,0 +1,48 @@
+# Typed Holes Request
```

Comment:
The description should include the level of stability we guarantee for this request. In this case, the level of stability is 0 and we will drop support for this request as soon as pulling diagnostics for a document is possible (as far as I understand, this will come in 3.17).